contents/blog/best-mobile-app-session-replay-tools.mdx

```diff
@@ -0,0 +1,346 @@
+</ComparisonTable>
+
+### How much does FullStory cost?
+
+FullStory doesn't publish pricing publicly. However, according to [Failory](https://www.failory.com/blog/fullstory-pricing), Fullstory will cost you between $250 - $1,000 per month.
+
+### Why do companies use FullStory?
+
+According to reviews on [G2](https://www.g2.com/products/fullstory/reviews), companies use FullStory for:
+
+1. **Easier collaboration:** As an accessible tool for non-technical users, FullStory facilitates collaboration between product, UX, and engineering teams by allowing all teams to access useful, reliable data.
+
+2. **Viewing user issues:** Support teams use FullStory to replay sessions to understand user hard-to-replicate problems, and identify bugs that need fixing.
+
+3. **Improving conversion:** FullStory users like to combine funnel insights with replays of user sessions to understand pain points and improve conversion.
+
+> #### Bottom line
+>
+> For a complete behavioral analytics suite, FullStory is a solid choice. Users wanting more features, like error tracking or a/b testing should switch to other choices. 
+
+<br />
+
+## 7. Contentsquare
+
+**Typical users:** Product managers, ecommerce teams, and marketers
+
+### What is Contentsquare?
+
+Contentsquare is an analytics platform that combines heatmaps, customer journey analysis, and frustration scoring to help find and fix issues and friction points.
+
+#### Supported features
+
+<ComparisonTable column1="Contentsquare">
+  <ComparisonRow column1={true} feature="Session replay" description="Watch real users on your mobile apps; discover friction points" />
+  <ComparisonRow column1={true} feature="Debug logs" description="View console logs, network logs, performance data" />
+  <ComparisonRow column1={true} feature="Filters" description="View specific replays based on user segments, screens visited, events" />
+  <ComparisonRow column1={true} feature="Heatmaps" description="See where users tap and interact" />
+  <ComparisonRow column1={true} feature="Scroll maps" description="See how far users scroll and where they dwell" />
+  <ComparisonRow column1={true} feature="Rage taps" description="Identify repetitive taps made out of frustration" />
+  <ComparisonRow column1={false} feature="Dead taps" description="Detect taps on elements that do not respond or trigger an action" />
+  <ComparisonRow column1={true} feature="Crash reports" description="Learn what happens before a crash without complex debugging or reproduction" />
+  <ComparisonRow column1={false} feature="AI summaries" description="Generate summaries of session replays" />
+</ComparisonTable>
+
+### How much does Contentsqaure cost?
+
+Since Contentsquare caters mostly to enterprise companies, their pricing is not available publicly and you need to speak to their sales team to get a quote. According to [Capterra](https://www.capterra.co.uk/software/161591/contentsquare), pricing starts at $10,000/month, but no further information is given
```

Comment:
```suggestion
Since Contentsquare caters mostly to enterprise companies, its pricing is not available publicly and you need to speak to its  sales team to get a quote. According to [Capterra](https://www.capterra.co.uk/software/161591/contentsquare), pricing starts at $10,000/month, but no further information is given
```